src/libs/Growl.js

```diff
@@ -4,13 +4,17 @@ import CONST from '../CONST';
 const growlRef = React.createRef();
 
 /**
- * Show the growl notification
+ * Show the growl notification. If the ref is not available, retry until it is.
  *
  * @param {String} bodyText
  * @param {String} type
  * @param {Number} [duration]
 */
 function show(bodyText, type, duration = CONST.GROWL.DURATION) {
+    if (!growlRef.current) {
+        setTimeout(() => show(bodyText, type, duration), 100);
+        return;
+    }
```

Comment:
Ah hmm are we trying to growl something before the growl mounts? What is it? Maybe we can just stop growling it.

Reply:
Calling [Growl.show](https://github.com/Expensify/App/blob/af0f42e2370179885981d6312cde7dd9cbb3ef1f/src/libs/actions/Policy.js#L130) after a policy is created is causing an error because Growl is undefined. Since there’s an error the promise is rejected and the user is never navigated away from the loading screen. The ref is initialized [here](https://github.com/Expensify/App/blob/main/src/Expensify.js#L188). [Here is a slack conversation](https://expensify.slack.com/archives/C03TQ48KC/p1647969014520139) about it. Removing the growl is definitely an option.

Reply:
Oh yeah, I remember running into this last time now. I got an idea for this actually that involves some code we already have implemented in the `ActiveClientsManager`. What do you think about this?

https://github.com/Expensify/App/pull/8271/files